src/bufr/BufrParser/BufrCollectors/BufrIntCollector.cpp

```diff
@@ -5,7 +5,7 @@
  * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-#include "bufr.interface.h"
+#include "bufr_interface.h"
```

Comment:
does this mean that this will only work with the latest BUFR develop branch now? Can we insure backwards and forwards compatibility somehow?

Reply:
Quick question (related to your request). What is the minimum version of compiler wee support for intel, gcc, micrsoft.. etc...

Reply:
The easiest way to do what you asked is by using the preprocessor __has_include method... but some older compilers might not support it.

Reply:
that would be a question for someone like @srherbener or @rhoneyager or @mmiesch maybe

Reply:
intel 17 and gcc 7.3 are the oldest we have built our stuff on.
I think the current stacks are probably atleast intel 18 and gnu 9.x

Reply:
Ryan is using it as part of the HH library, so maybe its ok.

Reply:
Support for __has_include was added in intel 18 (been part of gcc since version 5)

Reply:
Yes - you can count on intel 18+ and gcc 5+

Reply:
Thanks @mmiesch! I went ahead and used __has_include to check the header files. I think in the future we will be able to remove this so its only temporary.